### PR TITLE
Tutorial how to retrieve Bounding Boxes on Word Level of a Page

### PR DESCRIPTION
**Pull request by atraining**
This pull request introduces a new tutorial that provides step-by-step instructions on how to extract word-level bounding boxes (BBoxes) from a document using the Konfuzio SDK. The tutorial is in response to a user query about the availability of such a feature.

The tutorial includes the following:

1.  An explanation of how to import the necessary modules, initialize a project, and retrieve a document.
2.  Detailed instructions on how to use the WhitespaceTokenizer from the Konfuzio SDK to split the document into word-level spans.
3.  A guide on how to visualize all word-level annotations.
4.  A method to extract the bounding boxes for all word-level spans in the document.
5.  A section on how to preview the results by plotting the bounding boxes onto the original document image.

This feature aims to provide users with the ability to extract word-level BBoxes from their documents, which might be essential for certain use cases. The tutorial will be a valuable addition to our documentation, helping users understand and utilize this functionality more effectively.

### CODE REVIEW

**Comment by zypriafl** [comment]
This belongs to https://git.konfuzio.com/konfuzio/objectives/-/issues/11144#note_180286

**Review by iftwigs — COMMENTED** [review]
- a file name probably should be index.md
- we have the naming conventions set in glossary about the capitalization of our concepts https://github.com/konfuzio-ai/konfuzio-sdk/blob/master/docs/glossary.md

File: docs/sdk/examples/word-tokenizer/indext.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+document = deepcopy(document)
+```
+
+5. **Tokenize the document**:
+
+This process involves splitting the document into word-level spans using the WhitespaceTokenizer.
+
+```python
+tokenizer = WhitespaceTokenizer()
+document = tokenizer(document)
```
[the remark this answers is not in the document]
Comment (samuel500) [reply docs/sdk/examples/word-tokenizer/indext.md]: `document = tokenizer(document)` should work. I overwrote the `__call__` method in a recent PR: 
https://github.com/konfuzio-ai/konfuzio-sdk/blob/master/konfuzio_sdk/tokenizer/base.py#L212

[the remark this answers is not in the document]
Comment (iftwigs) [reply docs/sdk/examples/word-tokenizer/indext.md]: I checked myself when creating a testable code example, it does, thanks for the update!

**Review by samuel500 — APPROVED** [review]

**Pull request merged**